src/Aspire.Hosting.AWS/Utils/ProjectUtilities.cs

```diff
@@ -0,0 +1,123 @@
+using System.Runtime.InteropServices;
+using System.Text.Json;
+using System.Text.Json.Nodes;
+
+namespace Aspire.Hosting.AWS.Utils;
+
+internal static class ProjectUtilities
+{
+    /// <summary>
+    /// Initializes the project's launch settings if necessary, and
+    /// ensures they are referencing the Amazon.Lambda.TestTool's location.
+    /// </summary>
+    public static void UpdateLaunchSettingsWithLambdaTester(
+        string resourceName, 
+        string functionHandler, 
+        string assemblyName, 
+        string projectPath, 
+        string runtimeSupportAssemblyPath, 
+        string targetFramework)
+    {
+        // Retrieve the current launch settings JSON from wherever it's stored.
+        string launchSettingsJson = GetLaunchSettings(projectPath);
+
+        // Parse the JSON into a mutable JsonNode (root is expected to be an object)
+        JsonNode? rootNode = JsonNode.Parse(launchSettingsJson);
+        if (rootNode is not JsonObject root)
+        {
+            // If the parsed JSON isn’t an object, initialize a new one.
+            root = new JsonObject();
+        }
+
+        // Get or create the "profiles" JSON object
+        JsonObject profiles = root["profiles"]?.AsObject() ?? new JsonObject();
+        root["profiles"] = profiles;  // Ensure it's added to the root
+
+        var launchSettingsNodeKey = $"{Constants.LaunchSettingsNodePrefix}{resourceName}";
+
+        // Get or create the specific profile for Amazon.Lambda.TestTool
+        JsonObject? lambdaTester = profiles[launchSettingsNodeKey]?.AsObject();
+        if (lambdaTester == null)
+        {
+            lambdaTester = new JsonObject
+            {
+                ["commandName"] = "Executable",
+                ["executablePath"] = "dotnet"
+            };
+
+            profiles[launchSettingsNodeKey] = lambdaTester;
+        }
+
+        // Update properties that contain a path that is environment-specific
+        lambdaTester["commandLineArgs"] =
+            $"exec --depsfile ./{assemblyName}.deps.json --runtimeconfig ./{assemblyName}.runtimeconfig.json {SubstituteHomePath(runtimeSupportAssemblyPath)} {functionHandler}";
+        lambdaTester["workingDirectory"] = Path.Combine(".", "bin", "$(Configuration)", targetFramework);
```

Comment:
Somewhere there should be a check that we have version of RuntimeSupport in a targetFramework that the Lambda function is targeting. If we don't have a match we should log error and continue.